README.md

```diff
@@ -140,32 +140,6 @@ cd ..
 Check your first project
 ------------------------
 
```

Comment:
I think we should write down that the static analyzers are auto detected, and the latest available version is used for the analysis and if somebody wants to use a specific analyzer (custom build), how to configure CodeChecker.